### PR TITLE
Feature: Social Login

### PR DESCRIPTION
**Pull request by Ezuniga13**
Here can write some details....

### CODE REVIEW

File: file3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello
```
Comment (Ezuniga13) [inline file3.txt]: this line is good but better if you capitalize.

**Review by Ezuniga13 — COMMENTED** [review]
Please make these changes and do another push.

File: file3.txt (hunk 1 above)
Comment (EstebanZuniga13) [inline file3.txt]: I would all cap it.

**Review by EstebanZuniga13 — CHANGES_REQUESTED** [review]
Please make these changes and make another push.

**Review by EstebanZuniga13 — APPROVED** [review]
Everything looks perfect. Let's go ahead with merging.

**Pull request merged**